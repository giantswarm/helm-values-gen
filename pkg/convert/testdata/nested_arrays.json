{
  "$id": "https://example.com/person.schema.json",
  "$schema": "https://json-schema.org/draft/2020-12/schema",
  "properties": {
    "addresses": {
      "items": {
        "properties": {
          "city": {
            "type": "string",
            "default": "New York"
          },
          "state": {
            "type": "string",
            "default": "NY"
          },
          "street": {
            "type": "string",
            "default": "123 Main St"
          },
          "zip": {
            "type": "string",
            "default": "12345"
          }
        },
        "type": "object"
      },
      "type": "array"
    },
    "age": {
      "type": "integer",
      "default": 21
    },
    "firstName": {
      "type": "string",
      "default": "John"
    },
    "lastName": {
      "type": "string",
      "default": "Doe"
    }
  },
  "type": "object"
}
