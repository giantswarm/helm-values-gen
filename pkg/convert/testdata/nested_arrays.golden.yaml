age: 21
firstName: John
lastName: Doe
